README.md

```diff
@@ -1,6 +1,64 @@
-### MIND MAPP
+# MIND MAPP
 
-Mind Mapp: An interactive mind mapping tool for programmers
+```
+.        :   ::::::.    :::.:::::::-.  
+;;,.    ;;;  ;;;`;;;;,  `;;; ;;,   `';,
+[[[[, ,[[[[, [[[  [[[[[. '[[ `[[     [[
+$$$$$$$$"$$$ $$$  $$$ "Y$c$$  $$,    $$
+888 Y88" 888o888  888    Y88  888_,o8P'
+MMM  M'  "MMMMMM  MMM     YM  MMMMP"`
+
+.        :    :::.  ::::::::::.::::::::::.
+;;,.    ;;;   ;;`;;  `;;;```.;;;`;;;```.;;;
+[[[[, ,[[[[, ,[[ '[[, `]]nnn]]'  `]]nnn]]'
+$$$$$$$$"$$$c$$$cc$$$c $$$""      $$$""    
+888 Y88" 888o888   888,888o       888o     
+MMM  M'  "MMMYMM   ""` YMMMb      YMMMb   
+
+   an interactive mind mapping tool
+```
```

Comment:
I'm going to see if I can add Travis CI to this project so we can have a Travis badge also